README.md

```diff
@@ -77,20 +84,22 @@ source .venv/bin/activate
 ### Install PyTorch for your system
 
 If no explicit action is taken, the default PyTorch version will be installed.
-This will give you a current CUDA-based version. Install a different variant
-by doing so explicitly first:
+On Linux this may give you a current CUDA-based version. Install a different
+variant by doing so explicitly first, either by following the
+[official instructions](https://pytorch.org/get-started/locally/) or by using
+one of our `requirements.txt` files:
 
-*CPU:*
+* *CPU: [`pytorch-cpu-requirements.txt`](./pytorch-cpu-requirements.txt)*
```

Comment:
I think I'll convert these into absolute URLs so they continue to work at https://pypi.org/project/iree-turbine/